web/src/components/console/utils/status-box.tsx

```diff
@@ -20,26 +20,24 @@ const LoadError: React.FC<LoadErrorProps> = ({ label, className, message, canRet
     <Box className={className}>
       <div className="pf-u-text-align-center cos-error-title">
         {_.isString(message)
-          ? t('public~Error Loading {{label}}: {{message}}', {
+          ? t('Error Loading {{label}}: {{message}}', {
```

Comment:
We need o update all occurrences of `useTranslation` to add `'plugin__monitoring-plugin'`